### PR TITLE
[16.0][MIG] shopinvader_search_engine_assortment

### PR DESCRIPTION
**Pull request by sbejaoui**

### CODE REVIEW

**Comment by sbejaoui** [comment]
cc/ @lmignon

File: shopinvader_assortment/models/se_backend.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        product_obj = self.env["product.product"]
+        assortment_products = product_obj.search(domain)
+        product_indexes = self.index_ids.filtered(
+            lambda i: i.model_id.model == "product.product"
+        )
+        products_bound = (
+            product_indexes.binding_ids.record
+            if product_indexes.binding_ids
+            else product_obj
+        )
```
Comment (lmignon) [inline shopinvader_assortment/models/se_backend.py]: @sbejaoui  The use of the ORM for such a logic will introduce performance costs

Comment (lmignon) [reply shopinvader_assortment/models/se_backend.py]: @sbejaoui you can find [here](https://github.com/OCA/product-attribute/pull/1352/files#diff-71273ac18558d72e9ced9f26e9d84468a90f9bfd25d53c4ba9d3b85ff4540252R174) a way to write performant queries to get products to bind and to unbind....

**Comment by lmignon** [comment]
@sbejaoui IMO to be renamed in `shopinvader_search_engine_assortment`

**Review by lmignon — APPROVED** [review]
LGTM  (Code review)

File: shopinvader_search_engine_assortment/__manifest__.py [hunk 2]
```diff
@@ -2,17 +2,16 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
-    "name": "Shopinvader Assortment",
-    "version": "14.0.1.2.0",
+    "name": "Shopinvader Search Engine Assortment",
+    "version": "16.0.1.2.0",
```
Comment (simahawk) [inline shopinvader_search_engine_assortment/__manifest__.py]:
```suggestion
    "version": "16.0.1.0.0",
```

**Review by simahawk — APPROVED** [review]
tested, working fine. Minor remark

**Pull request merged**